app/views/payments/ChargeTypeDetailsView.scala.html

```diff
@@ -103,6 +103,28 @@ <h1 class="govuk-heading-xl" id="standard-charge-heading">
                 }
                 @model.title
             </h1>
+            <p class="govuk-body">
+                @if(!user.isAgent){
+                    @messages("chargeType.vatOverpayments.youNeedToPay")
+                } else {
+                   @messages("chargeType.vatOverpayments.agent.youNeedToPay")
+                }
+            </p>
+            <p class="govuk-body">@messages("chargeType.vatOverpayments.thisIsBecauseOf")</p>
+
+            <ul class="govuk-list govuk-list--bullet">
+                <li>
+                    @messages("chargeType.vatOverpayments.anError")
+                </li>
+                <li>@messages("chargeType.vatOverpayments.anOfficer")</li>
+                <li>
+                    @if(!user.isAgent){
+                     @messages("chargeType.vatOverpayments.aReallocation")
+                    } else {
+                     @messages("chargeType.vatOverpayments.agent.aReallocation")
+                    }
+                </li>
+            </ul>
```

Comment:
You need to check the charge type an only show this for the new charge type you've created